sessions/forecasting-models.qmd

```diff
@@ -31,6 +31,631 @@ The best way to interact with the material is via the [Visual Editor](https://do
 If not using the Visual Editor please remember that the code in the session needs to be run inside the course repository so that the `here()` commands below find the stan model files.
 :::
 
+# An overview of forecasting models
+
+See the slides for an overview of forecasting models.
+
+# What did we learn about the random walk model from the [forecasting concepts session](forecasting-concepts)?
+
+- It was unbiased when the reproduction number was constant but when the reproduction number was reducing due to susceptible depletion, the random walk model overestimated the number of cases systematically.
```

Comment:
was it really ever constant? I would have thought than on the log scale there is a similar amount of overestimation anywhere given more or less linear decline of R.